uktrade/data-hub-frontend
# Feature/routed dashboard

## Description of change

The dashboard tab navigation is now _routed_ i.e. it writes to and reads from the browser's address bar. 

## Test instructions

1. Open the homepage
2. The _My companies lists_ tab should be selected by default
3. Click the _My referrals_ tab
2. The path in the browser address bar should now be `/my-referrals`
3. Refresh the page with the `/my-referrals` path
4. After the page reloads, the _My referrals_ tab should be selected
5. Click on the _My companies lists_ tab
6. The path in the address barr should now read `/`
